language-plutus-core/test/types/reduce.plc

```diff
@@ -1,5 +1,5 @@
 (program 0.1.0
-  (lam x [(lam x (type) x) [(lam y (type) y) (con integer)]]
```

Comment:
Is this actually invalid according to the spec?

Reply:
This is the same issue as above, so I'm resolving it.